guides/common/modules/ref_system-requirements.adoc

```diff
@@ -101,7 +101,8 @@ SELinux must be enabled, either in enforcing or permissive mode.
 Installation with disabled SELinux is not supported.
 
 .FIPS Mode
-You can install {ProductName} on a {RHEL} system that is operating in FIPS mode.
+You can install {ProjectServer} on a {RHEL} system that is operating in FIPS mode.
```

Comment:
I think that you are not installing a project **server** on a system, but rather by installing **project** on the system, the system becomes the **project server**. 

```suggestion
You can install {Project} on a {RHEL} system that is operating in FIPS mode.
```